qiita_db/metadata_template/prep_template.py

```diff
@@ -136,6 +131,18 @@ def create(cls, md_template, study, data_type, investigation_type=None,
                 data_type_id = qdb.util.convert_to_id(data_type, "data_type")
                 data_type_str = data_type
 
+            # If the investigation_type is None let's add it based on the
+            # data_type being created - if possible
+            if investigation_type is None:
+                if data_type_str in {'16S', '18S', 'ITS'}:
```

Comment:
It might be good to do this as an enumerated type like we do in metapool and whatnot, but I can go either way.

Reply:
Do you mean like this: https://docs.python.org/3/library/enum.html ? If not, what do you mean? Also, I'm not sure of the benefits of going that route, could you expand?

Reply:
Under the [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) principle, unless this type of test is truly only done once ever

Reply:
Got it! ... and that comment made me remember TARGET_GENE_DATA_TYPES; changing for that.